src/AutoComplete/AutoComplete.component.tsx

```diff
@@ -125,9 +138,24 @@ export const AutoComplete = ({
     const [isFocused, setIsFocused] = useState<boolean>(false);
     // The current search term
     const [term, setTerm] = useState<string>(value ? `${value}` : '');
+    const autoFocusRef = useRef<HTMLDivElement>(null);
     // Instance of the nested input
     const inputRef = useRef<any>();
     const resultsRef = useRef<any>();
+    let rootElement: any;
+
+    // Handle click outside of auto complete component
+    const handleOutsideClick = (e: any) => {
+        if (isFocused && autoFocusRef.current && !autoFocusRef.current.contains(e.target)) {
+            setIsFocused(false);
+        }
+    }
+    // Set root element and listener for outside click
+    useEffect(() => {
+        rootElement = autoFocusRef.current ? autoFocusRef.current.getRootNode() : null;
+        rootElement && rootElement.addEventListener('click', handleOutsideClick);
```

Comment:
Lint CI is failing, likely due to this line. You should run `yarn lint` locally to get it resolved.